crates/ruff/src/rules/flake8_newspaper_style/rules/check_function_calls.rs

```diff
@@ -0,0 +1,169 @@
+use rustpython_parser::ast::{Expr, ExprKind, Located, StmtKind};
+
+use crate::checkers::ast::Checker;
+use crate::registry::AsRule;
+use ruff_diagnostics::{Diagnostic, Violation};
+use ruff_macros::{derive_message_formats, violation};
+use ruff_python_ast::types::{
+    Binding, BindingKind, ClassDef, FunctionDef, Range, RefEquality, Scope, ScopeKind,
+};
```

Comment:
I think this may need to be rebased, some of these got moved around the APIs changed a bit today.

Reply:
Yeah I'm already on it, however  it's taking me some time to understand some patterns in Rust 😅

Reply:
Let me know if I can help you with the rebase or if you have any questions

Reply:
Thanks :) I think I got it now but maybe you could check my new changes to see if I understood correctly